src/components/operations/operation-details/ko/runtime/authorization.html

```diff
@@ -0,0 +1,94 @@
+<!-- ko if: $component.authorizationServer() || $component.subscriptionKeyRequired() -->
+<h3 data-bind="css: { 'pad-top-0': $component.gqlConsole()}">Authorization 
+    <a href="#" role="button" data-bind="click: $component.collapseAuth">
+        <!-- ko if: $component.collapsedAuth -->
+        <i class="icon-emb icon-emb-chevron-down"></i>
+        <!-- /ko -->
+        <!-- ko ifnot: $component.collapsedAuth -->
+        <i class="icon-emb icon-emb-chevron-up"></i>
+        <!-- /ko -->
+    </a>
+</h3>
+
+<!-- ko ifnot: $component.collapsedAuth -->
+<!-- ko if: $component.authorizationServer -->
+<div class="row flex flex-row" data-bind="css: { 'col-6': $component.gqlConsole()}">
```

Comment:
`$component.gqlConsole()}` - I guess there should not be GQL specifics in this component.